tests/integration/validationTaksFlow/vlidationTaskFlow.helpers.ts

```diff
@@ -0,0 +1,96 @@
+import path from 'path';
+import { mkdir, readdir, rm } from 'fs/promises';
+import admZip from 'adm-zip';
+import { Feature } from 'geojson';
+import {} from '../../../src/common/constants';
+import { ChunkProcessor, ShapefileChunk, ShapefileChunkReader } from '@map-colonies/mc-utils';
+import { SHAPEFILE_REPORT_FILE_NAME, VALIDATION_ERROR_TYPE_FORMATS } from '../../../src/models/ingestion/constants';
+import { ErrorsCount } from '../../../src/models/ingestion/types';
+
+interface GetActualReportErrorsCountParams {
+  reader: ShapefileChunkReader;
+  reportDirPath: string;
+  jobId: string;
+}
+
+const validationErrorColumns = Object.values(VALIDATION_ERROR_TYPE_FORMATS).map((format) => format.columnName);
+
+const COLUMN_NAME_TO_COUNT_KEY: Record<string, keyof ErrorsCount> = Object.fromEntries(
+  Object.values(VALIDATION_ERROR_TYPE_FORMATS).map(({ columnName, countKey }) => [columnName, countKey])
+);
+
+const getReportFeatureCollection = async (reader: ShapefileChunkReader, reportPath: string): Promise<Feature[]> => {
+  const features: Feature[] = [];
+  const processor: ChunkProcessor = {
+    // eslint-disable-next-line @typescript-eslint/require-await
+    process: async (chunk: ShapefileChunk): Promise<void> => {
+      features.push(...chunk.features, ...chunk.skippedFeatures);
+    },
+  };
+  await reader.readAndProcess(reportPath, processor);
+
+  return features;
+};
+
+const findAndUnzipReport = async (reportDirPath: string, jobId: string): Promise<string> => {
+  const reportPath = reportPathBuilder(reportDirPath, jobId);
+  const files = await readdir(reportPath);
+  const zipFile = files.find((file) => file.endsWith('.zip'));
+
+  if (zipFile === undefined) {
+    throw new Error(`No zip file found in ${reportPath}`);
+  }
+
+  const zipPath = path.join(reportPath, zipFile);
+
+  const zip = new admZip(zipPath);
```

Comment:
in case of zip path is not exists, is it going to throw an error? should we add a check for this case?